libraries/3d-tools/app/layout.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<layout xmlns="https://geocortex.com/layout/v1" xmlns:custom="vertigis.web.incubator">
```

Comment:
Let's change the alias used here to be xmlns:incubator instead. This is one of the suggested changes made in the Time slider widget PR.

Reply:
updated